doc/source/cluster/aws-tips.rst

```diff
@@ -71,3 +71,226 @@ You should see something like
+
+The example config files above provide the following default configurations:
+
+1. Logs and Metrics:  Logs written to ``/tmp/ray/session_*/logs/**.out`` will be available in the ``{cluster_name}-ray_logs_out`` log group,
+and logs written to ``/tmp/ray/session_*/logs/**.err`` will be available in the ``{cluster_name}-ray_logs_err`` log group.
+Log streams are named after the EC2 instance ID that emitted their logs.
+Extended EC2 metrics including CPU/Disk/Memory usage and process statistics can be found in the ``{cluster_name}-ray-CWAgent`` metric namespace.
+
+2. Dashboard: You will have a cluster-level dashboard showing total cluster CPUs and available object store memory.
+Process counts, disk usage, memory usage, and CPU utilization will be displayed as both cluster-level sums and single-node maximums/averages.
+
+3. Alarms: Node-level alarms tracking prolonged high memory, disk, and CPU usage are configured. Alarm actions are NOT set,
+and must be manually provided in your alarm config file.
+
+For more advanced CloudWatch configurations options, refer to `agent <https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html>`_, `dashboard <https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/CloudWatch-Dashboard-Body-Structure.html>`_ and `alarm <https://docs.aws.amazon.com/AmazonCloudWatch/latest/APIReference/API_PutMetricAlarm.html>`_ AWS documentations.
+
+CloudWatch Agent, Dashboard, and Alarm JSON config files support the following variables:
+``{instance_id}``: Replaced with each EC2 instance ID in your Ray cluster.
+
+``{region}``: Replaced with your Ray cluster's region.
+
+``{cluster_name}``: Replaced with your Ray cluster name.
+
+See CloudWatch Agent `Configuration File Details <https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/CloudWatch-Agent-Configuration-File-Details.html>`_ for additional variables supported natively by the Unified CloudWatch Agent.
+
+.. note::
+    For CloudWatch alarm JSON configuration file, remember to add your alarm actions as shown below.
+
+.. code-block:: json
+
+     "AlarmActions":[
+         "TODO: Add alarm actions! See https://docs.aws.amazon.com/AmazonCloudWatch/latest/monitoring/AlarmThatSendsEmail.html"
+      ]
+
+3. Reference your own CloudWatch JSON configuration files as file path in cluster YAML config file.
+===================================================================================================
+
+You need to specify the file path to CloudWatch JSON configuration files relative to cluster YAML config file like:
```

Comment:
```suggestion
You need to specify the file path to your CloudWatch JSON config files relative to your cluster config YAML like:
```

Also, is this correct? Does it actually need to be relative to the cluster config YAML, or relative to the working directory that, say, `ray up config.yaml` is run from?

Reply:
This should be relative to your cluster config YAML run from instead, my bad, thanks for pointing it out!